File: threedollar-api/src/main/java/com/depromeet/threedollar/api/controller/MainController.java
package com.depromeet.threedollar.api.controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class MainController {

	@GetMapping("/ping")
	public ApiResponse<String> healthCheck() {
		return ApiResponse.SUCCESS;
	}

}

File: threedollar-api/src/main/java/com/depromeet/threedollar/api/controller/MainController.kt
package com.depromeet.threedollar.api.controller

import com.depromeet.threedollar.api.common.dto.ApiResponse
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class MainController {

    @GetMapping("/ping")
    fun healthCheck(): ApiResponse<String> {
        return ApiResponse.SUCCESS
    }

}
